plugins/blip/modalday/SMBGBox.js

```diff
@@ -11,6 +16,145 @@ d3.chart('SMBGBoxOverlay', {
     var boxPlotsGroup = this.base.insert('g', '#modalDays').attr('id', 'overlayUnderneath');
     var meanCirclesGroup = this.base.append('g').attr('id', 'overlayOnTop');
 
+    function getMsPer24(d) {
+      return dt.getMsPer24(d.msX);
+    }
+
+    /**
+     * Get the coordinates of the mean point given a record
+     * 
+     * @param  {Object} d the record
+     * @return {Array}   coordinates
+     */
+    function getMeanPosition(d) {
+      var mean = d3.select('#meanCircle-'+d.id);
+      return [mean.attr('cx'), mean.attr('cy')];
+    }
+
+    var tooltipHtml = function(tooltip, d) {
+      var rectOpts = chart.opts().infoRects;
+      var xScale = chart.xScale();
+      var yScale = chart.yScale();
+
+      var parent = d3.select(tooltip.tooltipGroup.node().parentNode);
+
+      var rangeLabels = parent.append('g')
+        .attr('class', 'smbgRangeInfo modalDay');
+
+      var maxYRect = yScale(d.max) - rectOpts.height - rectOpts.pad;
+      var maxYText = yScale(d.max) - (rectOpts.height/2);
+      var minYRect = yScale(d.min) + rectOpts.pad;
+      var minYText = yScale(d.min) + rectOpts.height;
+
+      if (d.max > 385) {
+        maxYRect = yScale(d.max) + rectOpts.pad;
+        maxYText = yScale(d.max) + rectOpts.height;
+      }
+
+      if (d.min < 60) {
+        minYRect = yScale(d.min) - rectOpts.height - rectOpts.pad;
+        minYText = yScale(d.min) - (rectOpts.height/2);
+      }
+
+      appendRangeLabel(rangeLabels, maxYRect, maxYText, rectOpts.width, 'max', d.max);
+      appendRangeLabel(rangeLabels, minYRect, minYText, (rectOpts.width - 5), 'min',d.min);
```

Comment:
Because its an SVG rect and overlay, this is the only way (that I know at least).